File: python3_solution_set2/2600. K Items With the Maximum Sum.py
class Solution:
    def kItemsWithMaximumSum(self, numOnes: int, numZeros: int, numNegOnes: int, k: int) -> int:

        arr = [1 for x in range(numOnes)] + [0 for x in range(numZeros)] + [-1 for x in range(numNegOnes)]

        return sum(arr[:k])

File: go_solution_set1/2600. K Items With the Maximum Sum.go
func kItemsWithMaximumSum(numOnes int, numZeros int, numNegOnes int, k int) int {
    ans := 0
    arr := []int{}

    for x:=0;x<numOnes;x++{
        arr = append(arr, 1)
    }
    for x:=0;x<numZeros;x++{
        arr = append(arr, 0)
    }
    for x:=0;x<numNegOnes;x++{
        arr = append(arr, -1)
    }
    for x:=0;x<k;x++{
        ans+=arr[x]
    }
    return ans 
}
